packages/fx-core/src/common/local/localEnvManager.ts

```diff
@@ -121,7 +121,7 @@ export class LocalEnvManager {
 
   public async getLocalEnvInfo(
     projectPath: string,
-    cryptoOption?: { projectId: string }
+    cryptoOption: { projectId: string }
```

Comment:
Why make this required? I remember for some cases just ignoring crypto to improve performance.

Reply:
Because this part uses ```environmentManager.loadEnvInfo``` to load envInfo, it needs ```crypto``` as a parameter.

Reply:
I remember the `crypto` in `loadEnvInfo` can be undefined. Does it still support undefined now?

Reply:
![image](https://user-images.githubusercontent.com/13211513/162384930-b8a5c68c-b2b6-489d-bcc2-c5b4ee87ed4d.png)
I have tried with undefined ```crypto```, it will fail in the end.

Reply:
Oh, I see. The original `LocalSettingsProvider` supports empty crypto, but the `LocalStateProvider` does not.